kurl_proxy/cmd/main.go

```diff
@@ -215,12 +227,16 @@ func getHttpsServer(upstream *url.URL, tlsSecretName string, secrets corev1.Secr
 	r.LoadHTMLGlob("/assets/*.html")
 
 	r.GET("/tls", func(c *gin.Context) {
+		if !acceptAnonymousUploads {
```

Comment:
If somebody manually enters `https://<ip>:8800/tls` then they will see the upload form, but then if they POST a cert/key it will 403. Is that the desired behavior?

Reply:
The intended behavior is that once certs have been uploaded, i.e. acceptAnonymousUploads is false, you can no longer get to the form.  You'll only be able to GET /tls again once this annotation is re-created.  I plan to document this in the kurl.sh docs.